main/solution/ui/src/i18next.js

```diff
@@ -0,0 +1,32 @@
+// this file should be imported only ONCE GLOBALlY, since there are init scripts.
+
+import i18next from 'i18next';
+import { initReactI18next } from 'react-i18next';
+import Backend from 'i18next-http-backend';
+import LanguageDetector from 'i18next-browser-languagedetector';
+
+import en from './locales/en.json';
+import zhCN from './locales/zh-CN.json';
+import placeholder from './locales/placeholder.json';
+
+i18next
+  // pass the i18n instance to react-i18next.
+  .use(initReactI18next)
+  .use(Backend)
+  .use(LanguageDetector)
+  .init({
+    resources: {
+      'en': en,
+      'zh-CN': zhCN,
+      'placeholder': placeholder,
+    },
+    lng: 'en', // if you're using a language detector, do not define the lng option
+    fallbackLng: 'placeholder',
```

Comment:
What's placeholder strings? Should we fallback to English by default?

Reply:
unnecessary files are removed from version control in the latest commit